frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -482,11 +481,7 @@ export class FieldValuesWidget extends Component {
           style={style}
           className={className}
           parameter={this.props.parameter}
-          optionsStyle={
-            optionsMaxHeight !== undefined
-              ? { maxHeight: optionsMaxHeight }
-              : {}
-          }
+          optionsStyle={!parameter ? { maxHeight: "none" } : {}}
```

Comment:
When there is no parameter `maxHeight` should be `none` https://github.com/metabase/metabase/pull/16308/files#diff-52dbad60907bc454d0a20c6b831ee0bb9d5e585d7ac0daab1f7b618f052d82a8R595